spec/intl.html

```diff
@@ -2489,8 +2465,17 @@ <h1>Temporal.ZonedDateTime.prototype.toLocaleString ( [ _locales_ [ , _options_
           <emu-alg>
             1. Let _zonedDateTime_ be the *this* value.
             1. Perform ? RequireInternalSlot(_zonedDateTime_, [[InitializedTemporalZonedDateTime]]).
-            1. Let _dateFormat_ be ? Construct(%DateTimeFormat%, « _locales_, _options_ »).
-            1. Return ? FormatDateTime(_dateFormat_, _zonedDateTime_).
+            1. Let _dateTimeFormat_ be ! OrdinaryCreateFromConstructor(%DateTimeFormat%, %DateTimeFormat.protoytpe%, « [[InitializedDateTimeFormat]], [[Locale]], [[Calendar]], [[NumberingSystem]], [[TimeZone]], [[Weekday]], [[Era]], [[Year]], [[Month]], [[Day]], [[DayPeriod]], [[Hour]], [[Minute]], [[Second]], [[FractionalSecondDigits]], [[TimeZoneName]], [[HourCycle]], [[Pattern]], [[BoundFormat]] »).
+            1. Let _timeZone_ be ? ToTemporalTimeZoneIdentifier(_zonedDateTime_.[[TimeZone]]).
+            1. If IsTimeZoneOffsetString(_timeZone_) is *true*, throw a *RangeError* exception.
+            1. If IsAvailableTimeZoneName(_timeZone_) is *false*, throw a *RangeError* exception.
+            1. Set _timeZone_ to CanonicalizeTimeZoneName(_timeZone_).
```

Comment:
Per my comment above, can we delegate all these steps to 402 ?

```suggestion
            1. Let _timeZone_ be _zonedDateTime_.[[TimeZone]].
```